install-cloud.sh

```diff
@@ -11,27 +11,28 @@
 
 set -e
 
-if [ -f /etc/centos-release ]
-then 
-  yum update -y
-  yum install -y git bind-utils nmap-ncat
```

Comment:
Can we leave the other centos-stuff in place?  If anyone's running their own VM anyways, it'd be nice to support more than ubuntu.